courses/admin.py

```diff
@@ -68,6 +69,32 @@ class CourseAdmin(admin.ModelAdmin):
         models.CharField: {"widget": TextInput(attrs={"size": "80"})}
     }
 
+    def get_readonly_fields(self, request, obj=None):
+        """
+        Adds `title` as readonly field while editing an existing object.
+        """
+        if obj:
```

Comment:
I have used `hasattr` as I was getting No attribute error when I was doing `obj.page`.

Reply:
Another more concise way you can do this to avoid the extra `and` is with `getattr(obj, "page", None) is not None`, because `getattr(None, "page", None) is None`. I'm nitpicking at this point so this is a take-it-or-leave-it kind of comment.